components/dictaphone/Dictaphone.js

```diff
@@ -6,9 +6,12 @@ import {
   Box,
   Box1,
   ContainerMain,
+  Mic,
   Navbar1,
   ScreenBox,
+  Microphone,
   ScreenBox1,
```

Comment:
Screenbox1 is an awful naming, are you also sure about it being a div?

Reply:
yes the component dictaphone comes with his display feature on the page screening to be manipulated